### PR TITLE
docs: :memo: update guides based on retrospectives

### PR DESCRIPTION
**Pull request by lwjohnst86**
## Description

This is a stacked PR. I've done a few updates to the docs based on what we've discussed in the last few retrospectives. I also rearranged and organized the "tasks during iteration" guide.

## Reviewer Focus

This PR needs an in-depth review.

## Checklist

- [x] Ran spell-check
- [x] Formatted Markdown
- [ ] Rendered website locally

### CODE REVIEW

**Review by K-Beicher — DISMISSED** [review]
Looks sensible to me

File: how-we-work/iteration-tasks.qmd [hunk 1]
```diff
@@ -20,13 +66,23 @@ Specifically:
...
+## Meetings
 
--   Respond to reviewer comments on our own pull requests.
--   Respond to issues where we are tagged/mentioned (`@`).
--   Review pull requests from others.
--   Work on the assigned highest priority issues.
--   Respond to comments and questions in issues, unless it is a
-    discussion issue, where the assigned team member will seek out and
-    request our comments.
+In general, we limit meetings as much as is necessary, however some can
```
Comment (martonvago) [inline how-we-work/iteration-tasks.qmd:71]:
```suggestion
In general, we limit meetings as much as possible, however some can
```
At least to me this makes more sense

File: how-we-work/iteration-tasks.qmd [hunk 2]
```diff
@@ -20,13 +66,23 @@ Specifically:
...
+    iteration/process), as well as to present a demo of the progress
+    made within the iteration.
+-   Schedule multi-person impromptu meetings as necessary and as
+    relevant for a given task or issue, for instance a [code review
+    meeting](code-review-meetings.qmd) or a [discussion
+    meeting](discussions.qmd).
+-   Plan meetings on knowledge sharing or for code reviewing when, for
+    instance, a new and more technically complicated feature is added or
+    a new tool is used in a pull request. For the author of the pull
+    request, they will need to do a bit of preparation before hand for
```
Comment (martonvago) [inline how-we-work/iteration-tasks.qmd:86]:
```suggestion
    request, they will need to do a bit of preparation beforehand for
```

File: how-we-work/iteration.qmd [hunk 3]
```diff
@@ -60,10 +60,11 @@ board](https://github.com/orgs/seedcase-project/projects/18/).
...
 
 An iteration starts with a first planning meeting that is scheduled well
-in advance, for between 1-2 hours and (preferably) right after the
-previous iteration's debrief and retrospective. Preferably this meeting
-is in person. The aim of the meeting to discuss and plan the specifics
-of the iteration.
+in advance, for between 30-45 minutes and either right after the
+previous iteration's debrief and retrospective or in the morning of the
+next day (to give a bit of break between meetings). Preferably this
+meeting is in person. The aim of the meeting to discuss and plan the
```
Comment (martonvago) [inline how-we-work/iteration.qmd:66]:
```suggestion
meeting is in person. The aim of the meeting is to discuss and plan the
```

File: how-we-work/iteration.qmd [hunk 4]
```diff
@@ -95,53 +95,26 @@ any potential issues as needed to complete the iteration aim.
...
-and the end date. We agree on and schedule the end date debrief and
-retrospective meeting, which should be approximately 2-3 hour and
-preferably before lunch, as well as the next iterations planning meeting
-(preferably on the same date, but after lunch to ensure a proper break
-before moving to a new iteration).
+decide or agree on the iteration goal (the "increment" or "milestone")
+and the end date.
+
+We'll review the current list of tasks on the board, brainstorm any
+other tasks that are needed, We'll then agree on the number of tasks in
```
Comment (martonvago) [inline how-we-work/iteration.qmd:102]:
```suggestion
We'll review the current list of tasks on the board and brainstorm any
other tasks that are needed. We'll then agree on the number of tasks in
```

**Review by martonvago — CHANGES_REQUESTED** [review]
Nice!! Just some language comments

**Review by K-Beicher — DISMISSED** [review]
Agree with Marton, looks good!

**Pull request merged**